src/List.ls

```diff
@@ -15,6 +15,13 @@ filter = (f, xs) -->
 reject = (f, xs) -->
   [x for x in xs when not f x]
 
+remove = (el, xs) -->
+  i = elem-index el, xs
+  results = xs.slice!
+  if i?
+    results.splice i, 1
+  results
```

Comment:
>but meh

Yup, it's a [bike shed](https://en.wikipedia.org/wiki/Bikeshedding). :) I'm happy either way.